data-loader/cli/src/main/java/com/scalar/db/dataloader/cli/ScalarDbMode.java

```diff
@@ -0,0 +1,16 @@
+package com.scalar.db.dataloader.cli;
```

Comment:
Should not be merged until https://github.com/scalar-labs/scalardb/pull/3147 is completed. Have to update the target branch before that.